app/models/user.rb

```diff
@@ -1,12 +1,38 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
There is no registeration or login form is shown when the app runs. The first page app shows is `users#index` page. I tried to create the user using console and it's created but devise is not functional. Kindly go through [this](https://github.com/heartcombo/devise) devise configuration, try to clone your project, and see that devise is not configure properly.

![index](https://user-images.githubusercontent.com/35504149/203617479-3501ef9e-6527-43a6-903a-f073993bc2f1.png)